test/fixture/composition-api/nuxt.config.js

```diff
@@ -0,0 +1,11 @@
+import { resolve } from 'path'
+import BaseConfig from '../base.config'
+
+/** @type {import('@nuxt/types').NuxtConfig} */
+const config = {
+  ...BaseConfig,
+  buildDir: resolve(__dirname, '.nuxt'),
+  srcDir: __dirname
+}
+config.modules?.push('@nuxtjs/composition-api')
```

Comment:
can you change to buildModules?
```suggestion
  srcDir: __dirname,
  buildModules: [
    '@nuxtjs/composition-api'
  ]
}

```

Reply:
Done